connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaConnectorConsumer.java

```diff
@@ -89,7 +89,7 @@ public KafkaConnectorConsumer(
   public void startConsumer() {
     if (elementProps.getAvro() != null) {
       var schemaString = StringEscapeUtils.unescapeJson(elementProps.getAvro().schema());
-      Schema schema = new Schema.Parser().setValidate(true).parse(schemaString);
+      Schema schema = new Schema.Parser().parse(schemaString);
```

Comment:
I assume the default constructor of the Parser sets the name validation to true:
See [here](https://github.com/apache/avro/blob/13c95b6249fb10c11ace01a50229b1ba6decca68/lang/java/avro/src/main/java/org/apache/avro/Schema.java#L1435)

But I am happy if someone else could also have a look at it